go.mod

```diff
@@ -7,6 +7,6 @@ require github.com/spf13/pflag v1.0.5
 require (
 	github.com/onsi/ginkgo v1.15.2
 	github.com/onsi/gomega v1.11.0
-	k8s.io/apimachinery v0.20.2
-	sigs.k8s.io/kubebuilder/v3 v3.0.0-beta.1
+	k8s.io/apimachinery v0.21.2
+	sigs.k8s.io/kubebuilder/v3 v3.1.0
```

Comment:
can you bump this to the latest commit of kb https://github.com/kubernetes-sigs/kubebuilder/commit/742983631190812ce81402fb662998fe07447e21

Reply:
After you bump this, ginkgo and gomega versions would also be bumped.

Reply:
Yes Modified it.

Reply:
ginko and gomega version also got updated.